js/src/main/scala/Main.scala

```diff
@@ -346,8 +315,14 @@ object Main {
           results append S(d.nme) -> (getType(ty_sch).show)
         case d @ Def(isrec, nme, R(PolyType(tps, rhs))) =>
           val errProv = TypeProvenance(rhs.toLoc, "def signature")
-          val ty_sch = PolymorphicType(0, typeType(rhs)(ctx.nextLevel, raise,
-            vars = tps.map(tp => tp.name -> freshVar(noProv/*FIXME*/)(1)).toMap))
```

Comment:
Dude... 🙄

Reply:
The change has been reverted. I’m making scalafmt ignore this file so it won’t happen again.

Reply:
Yes. They politely refused my request. But anyway, now I’m reducing my use of formatter.

Reply:
Is this another issue or the same as the one above? Link?